.circleci/config.yml

```diff
@@ -482,24 +494,28 @@ jobs:
   django:
     <<: *machine_executor
     steps:
+      - attach_workspace:
+          at: .
       - checkout
       - run: SNAPSHOT_CI=1 docker-compose up -d memcached redis testagent
       - run:
           command: docker-compose logs -f
           background: true
       - run:
-          command: ./scripts/ddtest scripts/run-tox-scenario '^django_contrib-'
+          command: "./scripts/ddtest riot -v run -s 'django$'"
 
   djangorestframework:
-    executor: ddtrace_dev
-    docker:
-      - image: *ddtrace_dev_image
-      - image: *redis_image
-      - image: *memcached_image
-      - *datadog_agent
+    <<: *machine_executor
     steps:
-      - run_tox_scenario:
-          pattern: '^django_drf_contrib'
+      - attach_workspace:
+          at: .
+      - checkout
+      - run: SNAPSHOT_CI=1 docker-compose up -d memcached redis testagent
+      - run:
+          command: docker-compose logs -f
+          background: true
+      - run:
+          command: "./scripts/ddtest riot -v run -s 'djangorestframework'"
```

Comment:
Should we use something like `run_test` for docker builds to abstract this part away?

Reply:
> I wonder if there is a way to validate that the Riotfile was updated to run the same scenarios as `tox`?

Updated PR with listing of 69 cases across riot and tox.

Reply:
> Should we use something like `run_test` for docker builds to abstract this part away?

Yes, definitely think that makes sense but decided to leave that for the next integration we migrate, but can update it here if necessary.

Reply:
hehe, or do a starting PR to abstract the docker specific stuff.... then this PR for django :p